### PR TITLE
Week 4 Charts Walkthrough

### PR DESCRIPTION
**Pull request by mnuzen**
Adding static chart feature on website with Google Charts

### CODE REVIEW

**Review by arunk054 — COMMENTED** [review]
It looks like you forked week4 off of week3 instead of master. I understand this is because of my delay in review, can you now try to merge the week3 to master and then do a rebase of week4 on to master?

**Comment by mnuzen** [comment]
week3 merged into master now

File: portfolio/src/main/java/com/google/sps/servlets/WebDataServlet.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    response.setContentType("application/json");
+    Gson gson = new Gson();
+    String json = gson.toJson(webVotes);
+    response.getWriter().println(json);
+  }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
```
Comment (arunk054) [inline portfolio/src/main/java/com/google/sps/servlets/WebDataServlet.java:42]: Add a comment for each API on what it does. Does this increment the votes?

File: portfolio/src/main/java/com/google/sps/servlets/WebDataServlet.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    String web = request.getParameter("web");
+    int currentVotes = webVotes.containsKey(web) ? webVotes.get(web) : 0;
```
Comment (arunk054) [inline portfolio/src/main/java/com/google/sps/servlets/WebDataServlet.java:44]: You could use the getOrDefault API instead.
 https://www.geeksforgeeks.org/hashmap-getordefaultkey-defaultvalue-method-in-java-with-examples/

**Review by arunk054 — APPROVED** [review]

**Pull request merged**